src/pages/login/login.ts

```diff
@@ -0,0 +1,20 @@
+import { Component } from '@angular/core';
+
+import { NavController } from 'ionic-angular';
+import { TabsPage } from '../tabs/tabs';
+
+@Component({
+  selector: 'page-login',
+  templateUrl: './login.html'
+})
+export class LoginPage {
+
+  constructor(public navCtrl: NavController) {
```

Comment:
Yes, it initializes the NavController by having it as a parameter. It makes NavController publicly accessible in the class through `navCtrl`.